src/xnatdcat/cli_app.py

```diff
@@ -182,6 +196,13 @@ def run_cli_app():
 
     config = load_configuration(args.config)
 
+    if args.optin:
```

Comment:
do you really need to check if args.optin/out here? Default is None so you can just assign args.optin/out with corresponding config key and let argparse to check everything else

Reply:
The keywords can be set in config, but CLI takes precedence over config file. So need to check they are set, else I'd overwrite the config with "None". I will rewrite it to make it one statement.

Reply:
OK, thank you for the clarification